modules/rosa-classic-aws-rhcos-ami.adoc

```diff
@@ -0,0 +1,969 @@
+|`rhcos-46.82.202008181646-0`
+|`ami-034e60a30bf173ea5`
+|`rhcos-42.80.20191002.0-hvm`
+|`ami-00f8b34bbeefddddd`
+|`rhcos-45.82.202008010929-0`
+|`ami-07adb9340bf130666`
+|`rhcos-44.82.202008011133-0-hvm`
+|`ami-0b059d767817b443f`
+|`rhcos-416.94.202403071059-0-x86_64`
+|`ami-078a02880a7f28fb9`
+|`rhcos-48.84.202107202156-0`
+|`ami-0ded752197186c01e`
+|`rhcos-47.83.202102090044-0`
+|`ami-0387f83845eb1a32b`	
+|`[Copied ami-05f09360cb1869045 from us-east-1] rhcos-44.81.202007071934-0-hvm`
+|`ami-0eeb9e8f6ede0099c`
+|`rhcos-410.84.202111201002-0-x86_64`
+|`ami-05c3f2ad7963594f3`
+|`rhcos-414.92.202305090606-0-x86_64`
+
+.19+|*eu-south-1*
+|`ami-07dc6682b8de235a8`
+|`rhcos-42.80.20191002.0-hvm`
+|`ami-02e6ff7df83e20271`
+|`rhcos-48.84.202107202156-0`
+|`ami-09885f805b18b7542`
+|`rhcos-49.84.202107191624-0`
+|`ami-06bc4f03cca2834f6`
+|`rhcos-48.84.202105062123-0`
+|`ami-06d5068b959daeef1`
+|`rhcos-412.86.202207142104-0-x86_64`
+|`ami-09b8d42580e13dfbb`
+|`rhcos-411.85.202203242008-0-x86_64`
+|`ami-037df250c060f1a85`
+|`rhcos-414.92.202305090606-0-x86_64`
+|`ami-06f41f191c93c2473`
+|`rhcos-47.83.202102090044-0`
+|`ami-05cb3152ba1bfea18`	
+|`[Copied ami-05f09360cb1869045 from us-east-1] rhcos-44.81.202007071934-0-hvm`
+|`ami-00e0eb96b5dace392`
+|`rhcos-45.82.202008010929-0`
+|`ami-06b19117e3e71b580`
+|`rhcos-410.84.202111201002-0-x86_64`
+|`ami-02b5434947e699db9`
+|`rhcos-416.94.202403071059-0-x86_64`
+|`ami-0f42b5d7603ace8c5`
+|`rhcos-415.92.202309201557-0-x86_64`
+|`ami-039021677b936e7d1`
+|`rhcos-413.86.202301302144-0-x86_64`
+|`ami-06f9ed4481d613bc6`
+|`rhcos-45.81.202005200134-0`
+|`ami-028d06116fead2ccf`
+|`rhcos-43.82.202008010953.0-hvm`
+|`ami-02c5c09b4203097b2`
+|`rhcos-46.82.202008071940-0`
+|`ami-05c9105cc61320b72`
+|`rhcos-44.82.202008011133-0-hvm`
+|`ami-06191e0b43a6bfacf`
+|`rhcos-46.82.202008181646-0`
+
+.19+|*eu-west-3*
+|`ami-068d3dd3c638bd157`
+|`rhcos-46.82.202008071940-0`
+|`ami-0f5ad1dbc8722f7dc`
+|`rhcos-42.80.20191002.0-hvm`
+|`ami-0c3a1026ac3f99cca`
+|`rhcos-413.86.202301302144-0-x86_64`
+|`ami-06d868044e94b4c3d`
+|`rhcos-48.84.202107202156-0`
+|`ami-0d5f2c4aaf3302d7e`
+|`rhcos-45.81.202005200134-0`
+|`ami-0eee03b7f795ca880`
+|`rhcos-48.84.202105062123-0`
+|`ami-0b17362653f2397bf`
+|`rhcos-49.84.202107191624-0`
+|`ami-0f581d9ec4d7db141`
+|`rhcos-411.85.202203242008-0-x86_64`
+|`ami-075ebd9414a260c34`
+|`rhcos-46.82.202008181646-0`
+|`ami-05967a4e65a8356d9`
+|`rhcos-414.92.202305090606-0-x86_64`
+|`ami-04d292a1e2821f359`
+|`rhcos-412.86.202207142104-0-x86_64`
+|`ami-0cc15eae9e71c3d26`
+|`rhcos-47.83.202102090044-0`
+|`ami-0e6a6a923427e76a6`
+|`rhcos-43.82.202008010953.0-hvm`
+|`ami-0d1987bdab0cc7903`
+|`rhcos-44.82.202008011133-0-hvm`
+|`ami-063c6f9a727602a3b`
+|`rhcos-410.84.202111201002-0-x86_64`
+|`ami-0e2a55691789771d0`
+|`rhcos-416.94.202403071059-0-x86_64`
+|`ami-09b8632b92e5edac5`
+|`rhcos-45.82.202008010929-0`
+|`ami-074c99ae8530e4eb3`	
+|`[Copied ami-05f09360cb1869045 from us-east-1] rhcos-44.81.202007071934-0-hvm`
```

Comment:
Can we clarify why the `[Copied ami-.... from us-east-1]` is there? Can we remove it?